dipy/viz/tests/test_util.py

```diff
@@ -69,3 +70,16 @@ def test_check_img_dtype():
         check_img_dtype(images)
         check_for_warnings(l_warns, 'skipping image 1, passed image is not in '
                            + 'numerical format')
+
+
+def test_is_binary_image():
```

Comment:
Serge's comment on random also applies here. Looking at some test files (e.g. dipy/core/tests/test_gradients.py) should tell you what you need to do